saleor/checkout/actions.py

```diff
@@ -239,3 +244,12 @@ def transaction_amounts_for_checkout_updated(
             checkout_info=checkout_info,
             lines=lines,
         )
+
+    channel = checkout_info.channel
+    if (
+        channel.automatically_complete_fully_paid_checkouts
```

Comment:
Just to be aware: The current implementation allows you to trigger multiple tasks for the same checkout. 
For example:
- You receive a transaction that caused that the checkout is fully-authorized
- You triggered the `automatic_checkout_completion_task`
- Before celery pick up the task, the new transaction for the same checkout was created
- if the `transaction_amounts_for_checkout_updated` will be called, the second celery task will be placed on the queue. 
(this is why a few lines above there is an if `if not previous_charge_status_is_fully_paid and current_status_is_fully_paid:` for charge and triggering the fully_paid event.)

In the worst scenario, this would cause `checkout_complete` will be called for the same checkout at the similar time. Conversion is locked so we should be safe, but maybe worth confirming that we're calling this event only once.